packages/page-template-modal/package.json

```
@@ -28,34 +28,26 @@
 		"src"
 	],
```

other typescript packages include the "types" root in `package.json`

```suggestion
	],
	"types": "dist/types",
```

---

Ah yes, this is important! I missed that and would have seen it if I'd actually converted some of `starter-page-templates` to TS as well like you suggest.
I'll do that.